aldryn_client/utils.py

```diff
@@ -231,6 +236,20 @@ def get_git_commit():
             pass
 
 
+def get_git_checked_branch():
```

Comment:
There's a more direct command to getting the current branch (which also doesn't need any custom string parsing in python): 

```bash
$ git rev-parse --abbrev-ref HEAD
master
```

It's available since Git 1.6.3 which was released in May, 2009 - that should be old enough for most distros. More info:

* https://github.com/git/git/releases/tag/v1.6.3
* http://stackoverflow.com/a/1418022